tests/installation/addon_products_sle.pm

```diff
@@ -45,6 +45,10 @@ sub handle_all_packages_medium {
     # According to bsc#1089455, migrating from SLED12 to SLED15, legacy shouldn't be selected by the testcase.
     push @addons, 'legacy' if get_var('UPGRADE') && !grep(/^legacy$/, @addons) && !check_var('SLE_PRODUCT', 'sled');
 
+    # The development-tool module is required if upgrade from previous version
+    # Refer to https://fate.suse.com/325293
+    push @addons, 'sdk' if get_var('UPGRADE') && !grep(/^sdk$/, @addons) && !check_var('SLE_PRODUCT', 'sled');
```

Comment:
According to fate, `sdk` is enabled automatically by `sles, rt, hpc`, so not `sled` is not very precise